code/src/components/ToDos.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components/macro'
+import moment from 'moment'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrashAlt } from '@fortawesome/free-solid-svg-icons'
+
+import todos from '../reducers/todos'
+import Image from '../components/Image'
+
+const ContainerDiv = styled.div `
+  height: 57vh;
+  max-width: fit-content;
+  margin-top: -50px;
+  overflow-y:auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  & img {
+    opacity: 0.5;
+  }
+
+  @media (min-width: 667px) {
+    margin-top: auto;
+  }
+`
+const StyledForm = styled.form `
+  width: 80vw;
+  max-width: 340px;
+  margin-bottom: 10px;
+  padding: 5px;
+  font-size: 20px;
+  border: 1px solid var(--red);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  border-radius: 5px;
+  opacity: ${props => props.completed ? 0.3 : 1};
+  background-color: ${props => props.category === 'Study' ? 'var(--lax)' : props.category === 'Work' ? '#6997b0' : props.category === 'Shopping' ? '#9e9dd9' : '#facdc5'};
```

Comment:
That one was gold! I really love styled-components now and what they can do!

Reply:
I am looking forward to see some cool styling ideas in your future projects! 😺